tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -414,3 +440,42 @@ def test_vm_snap_of_clone(
             vm_clone_fixture,
             vm_snapshot_restore_fixture,
         )
+
+    @tier2
+    @pytest.mark.polarion_id("")
+    def test_clone_of_restored_vm(
```

Comment:
can't we combine this test with https://github.com/red-hat-storage/ocs-ci/blob/master/tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py#L181 adding additional steps to  Clone the restored VM at the end?

Reply:
hi Prasad , We have not planned such scenario in testcases. clone- restore vm - clone. 
we decided to go with separate scenario. so I have added new testcase.
This is just enhancement of Ayush's opened PR.